jira/client.py

```diff
@@ -3165,6 +3171,9 @@ def create_board(self, name, project_ids, preset="scrum"):
             project_ids = project_ids.split(',')
         payload['projectIds'] = project_ids
         payload['preset'] = preset
+        if self.deploymentType == 'Cloud':
+            payload['locationType'] = location_type
```

Comment:
These are now required parameters in cloud when using a preset for an existing project.